src/client/js/view/options.js

```diff
@@ -148,11 +148,6 @@ class OptionsView extends BaseView {
       let $bookmark = $(e.currentTarget);
       let selectedDatasets = this.getSelectedDatasets();
 
-      // Do not allow deselect if only one dataset is selected.
-      if (selectedDatasets.length == 1 && $bookmark.hasClass('selected')) {
-        return;
-      }
```

Comment:
Q. Are we sure we want to remove this? What elements have the `bookmark` class attached to them? Perhaps the best solution would be to change where the bookmark class is instead of delete this code block?